package.json

```diff
@@ -13,6 +13,7 @@
   },
   "dependencies": {
     "@nightwatch/chai": "5.0.2",
+    "@nightwatch/mobile-helper": "^0.1.4",
```

Comment:
We are now installing this in `create-nightwatch`, so we can safely remove it from here.

@beatfactor should we add it to `devDependencies`?

Reply:
Removed it. I don't think adding this to devDependencies adds any value